iacrcc/iacrdoc.tex

```diff
@@ -65,16 +65,12 @@
 
 \begin{abstract}
   This document is a quick introduction to the \texttt{iacrcc.cls}
-  \LaTeX{} class for the \publname{}.
+  \LaTeX{} class for the IACR Communications in Cryptology.
 \end{abstract}
 
 \begin{textabstract}
-This is a quick introduction to the iacrcc.cls LaTeX class for
-The IACR Communications in Cryptology. No macros are allowed in
-the textabstract, but you may use unicode such as Paul Erdős, or
-diacriticals like F\"ur Elise.
-
-Multiple paragraphs are allowed.
+  This document is a quick introduction to the iacrcc.cls
+  LaTex class for the \publname{}.
```

Comment:
the use of \publname{} is precisely the kind of thing that is illegal inside the `textabstract` environment. No user-defined macros are allowed there. I'm not sure why you switched them between abstract and textabstract, because they were right the original way.

Reply:
ooops, you are right. That is a copy / paste mistake on my end.